lib/neo4j/migrations/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Neo4j
+  module Migrations
+    module Helpers
+      def rename_property(label, old_property, new_property)
+        by_label(label).where(n: {new_property => nil})
```

Comment:
@cheerfulstoic Maybe I could fail by default if there's some new_property `nil`, and add a parameter `force`?